public/js/null_plugins/report_plugins.js

```diff
@@ -16,7 +17,12 @@ class ReportPlugins {
     }
 
     generateStats() {
-        return (<Histogram />);
+        var colors = chroma.scale('RdYlBu').colors(1);
+        return (
+          <div className='histogram-container'>
+            <Histogram query="evalue" name="E-Value Distribution" color={colors[0]} />
+          </div>
```

Comment:
Would it be possible to clean up this PR to only inlcude the performance fix without the Histogram changes?

Reply:
can we keep it?

this PR is required to fix performance before we merge histogram.